demo/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "react-accordion-demo",
+  "version": "1.0.0",
+  "description": "A really lightweight, symantic, super simple accordion for React",
+  "type": "module",
+  "scripts": {
+    "develop": "servatron -d dist & esbuild --watch --bundle src/index.tsx --outfile=dist/index.min.js --loader:.js=jsx",
+    "build": "esbuild --bundle src/index.tsx --outfile=dist/index.min.js --loader:.js=jsx"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/votemike/react-accordion.git"
+  },
+  "keywords": [
+    "accordion",
+    "accessible",
+    "react",
+    "ui"
+  ],
+  "author": "",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/votemike/react-accordion/issues"
+  },
+  "homepage": "https://github.com/votemike/react-accordion#readme",
+  "dependencies": {
+    "react": "^18.2.0",
+    "react-accordion": "../"
+  },
+  "devDependencies": {
```

Comment:
When does the demo have dev dependencies?

Reply:
Not sure I understand the question. You can see them below?

Reply:
Is the demo just for use locally? If so, why do any of the dependencies need to be dev?

Reply:
Think of `demo` as it's own [packgage](https://docs.npmjs.com/cli/v7/using-npm/workspaces) or project in itself. The demo should also be a reflection of a project using the react-accordion, of which you would expect the dependencies to be separated into their proper scope.